lib/suspenders/app_builder.rb

```diff
@@ -322,6 +318,7 @@ def gitignore_files
       copy_file 'suspenders_gitignore', '.gitignore'
       [
         'app/views/pages',
+        'app/workers',
```

Comment:
did thoughtbot forget that gitignore files is a thing already? maybe thats an upstream contribution?